strapi/strapi
# Updated notifications container position to fixed

**My PR is a:**
- [ ] 🐛 Bug fix #issueNumber

**Main update on the:**
- [ ] Framework


**Manual testing done on the following databases:**
- [ ] MongoDB


On smaller screens when you scroll down, the current notification is positioned at the top and you will not see the notifications.

To replicate the issue, simply trigger strapi.notification.info('This is a strapi notification') and observe the difference when you scroll down
